sycl/test/basic_tests/image_accessor_readsampler.cpp

```diff
@@ -1,4 +1,5 @@
 // UNSUPPORTED: cuda
```

Comment:
is this test still failing on CUDA? If so could you provide more detail description for TODO?

Reply:
Comment from line 2: `// CUDA cannot support SYCL 1.2.1 images.` 
Would you like to see anything else?